packages/api/src/api-manager.ts

```diff
@@ -220,15 +226,16 @@ export class ApiManager implements ApiManagerInterface {
           '[project_id]',
           this._projectId.toString()
         ),
-        route: `/track/${this._accountId}/${this._projectId}`
+        route: `/track/${this._sdkKey}`
```

Comment:
was the metrics endpoint updated to support this?(I forgot)

Reply:
No idea!

Reply:
oh, I wasnt even aware that nestJs exist, I only new about nextJs :))

Reply:
We're all good, I had the SDK looking for the `sdkKey` in config, if not found, fallback to `accountId/projectId`:
```typescript
this._sdkKey = config?.sdkKey || `${this._accountId}/${this._projectId}`;
```